dashboard/test/ui/features/weblab/versions.feature

```diff
@@ -0,0 +1,44 @@
+@dashboard_db_access
+@as_student
+Feature: Weblab Versions
+
+Background:
+  Given I am on "http://studio.code.org/s/allthethings/stage/32/puzzle/1?noautoplay=true"
+  Then I rotate to landscape
+  And I wait until element "#submitButton" is visible
+
+Scenario: Weblab Versions
+  # Create the initial version by adding a CSS file:
+  When I click selector "#headers div:nth-child(2)"
+  And I wait for 3 seconds
```

Comment:
We usually try to avoid arbitrary wait times in UI tests because they're more brittle than waiting for a particular element to appear.  I understand that might not be possible in this test due to the cross-origin issues - is there anything we can watch for at this step and the one below?

Reply:
Yes, it is definitely tough to wait on something that we can't see. If we figure out cross-domain access, it will help UI tests for weblab for sure!